build.gradle

```diff
@@ -9,9 +9,6 @@ plugins {
 group 'com.github.hmcts'
 version 'DEV-SNAPSHOT'
 
-sourceCompatibility = 11
```

Comment:
Do we need this to specify 17? Any reason we don't?

Reply:
I tried specifying it to 17, but throws "error: invalid source release: 17",